### PR TITLE
Eilenberg-Moore adjunction

### PR DESCRIPTION
**Pull request by anuyts**
Builds on #764.
Result: `FreeEMAlgebra -| ForgetEMAlgebra`.

### CODE REVIEW

File: Cubical/Categories/Instances/EilenbergMoore.agda
Comment (mortberg) [inline Cubical/Categories/Instances/EilenbergMoore.agda:89]: What is this commented code?

File: Cubical/Categories/Instances/EilenbergMoore.agda
Comment (mortberg) [inline Cubical/Categories/Instances/EilenbergMoore.agda:154]: Long line

**Review by mortberg — CHANGES_REQUESTED** [review]
I think I only spotted one more long line and a comment that could be removed. Apart from this I'd be happy to merge once https://github.com/agda/cubical/pull/764 is merged

File: Cubical/Categories/Instances/EilenbergMoore.agda
Comment (anuyts) [reply Cubical/Categories/Instances/EilenbergMoore.agda]: Ah sorry, a failed attempt to prove the other formulation of adjointness.

**Comment by mortberg** [comment]
And now there's some conflicts. Please fix and I'll merge

File: Cubical/Categories/Constructions/FullSubcategory.agda
Comment (mortberg) [inline Cubical/Categories/Constructions/FullSubcategory.agda:15]: This comment can be removed

**Review by mortberg — CHANGES_REQUESTED** [review]
one small comment and some conflicts before I can merge

**Pull request merged**
